manage.py

```diff
@@ -173,6 +246,15 @@ def stop(args):
         print_color(f"Stopping kind cluster", bcolors.OKBLUE)
         run_shell(f"kind delete cluster")
 
+    elif (f"{args.platform}" == "EKS"):
+        """
+        Shutdown the EKS Cluster with the demo environment deployed.
+        """
+        print_color(f"Stopping EKS cluster with name {args.name}", bcolors.OKBLUE)
+
+        # Stop cluster
+        run_shell(f"eksctl delete cluster --name={args.name}")
```

Comment:
We should also have the support of the `--region` flag with stop

Reply:
Updated command to accept AWS region input